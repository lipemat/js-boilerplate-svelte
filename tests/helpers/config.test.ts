describe( 'config.js', () => {
	test( 'Default TSConfig', () => {
		const {getTypeScriptConfig} = require( '../../helpers/config.js' );
		expect( getTypeScriptConfig() ).toMatchSnapshot();
	} );
} );
