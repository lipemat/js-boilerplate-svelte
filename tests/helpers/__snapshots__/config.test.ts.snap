// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config.js Default TSConfig 1`] = `
{
  "$schema": "https://json.schemastore.org/tsconfig",
  "_version": "5.0.0",
  "compilerOptions": {
    "esModuleInterop": true,
    "forceConsistentCasingInFileNames": true,
    "module": "esnext",
    "moduleResolution": "bundler",
    "skipLibCheck": true,
    "sourceMap": true,
    "strict": false,
    "target": "es2017",
    "verbatimModuleSyntax": true,
  },
  "display": "Svelte",
}
`;
